scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -547,6 +548,57 @@ static u64 sched_prio_to_latency_weight(u64 prio)
 	return sched_prio_to_weight[DL_MAX_LAT_PRIO - prio - 1];
 }
 
+
+/*
+ * Returns the preferred domain according to the mempolicy. See man(2)
+ * set_mempolicy for more details.
+ */
+static u32 task_pick_mempolicy_domain(
+		struct task_struct *p, struct task_ctx *taskc, u32 rr_token)
+{
+	u32 ret = NO_DOM_FOUND;
+	if (!mempolicy_affinity || !bpf_core_field_exists(p->mempolicy) ||
+			!p->mempolicy || !taskc->cpumask)
+		return ret;
+
+	if (!(p->mempolicy->mode & (MPOL_BIND|MPOL_PREFERRED))) {
```

Comment:
Last nit (sorry for all the nits): I know it's annoying, but in kernel coding style, single-line conditionals don't use the curly braces